nvidia-runtime.sysext/files/usr/local/bin/_nvidia-persistenced-helper

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+set -euo pipefail
+
+SYSEXT_MODE=0
+if systemd-sysext list 2>&1 | grep -q flatcar-nvidia-drivers; then
+  SYSEXT_MODE=1
+fi
+
+_pre() {
+  if [ "$SYSEXT_MODE" -eq 0 ]; then
+    rm /run/extensions/nvidia-driver && systemctl restart systemd-sysext || true
+  fi
+}
+
+_post() {
+  chcon -R -t container_file_t /dev/nvidia* || true
+
+  if [ "$SYSEXT_MODE" -eq 0 ]; then
+    /opt/bin/nvidia-persistenced || true
+    mkdir -p /run/extensions
+    # /opt/nvidia/current gets mounted as an overlay on root filesystem
+    mkdir -p /opt/nvidia/current/usr/bin && ln -sf /opt/bin/nvidia-smi /opt/nvidia/current/usr/bin/nvidia-smi
+    mkdir -p /opt/nvidia/current/usr/local/ && ln -sf /opt/nvidia /opt/nvidia/current/usr/local/nvidia
```

Comment:
were you planning on moving these lines to the `nvidia-driver.service`? Since it's only in that case that we mount `/opt/nvidia/current` as a sysext.

Second question: does sysext_mode ship `/usr/local/nvidia`?

Reply:
`/usr/local/nvidia` appears to be used when we request that the GPU -Operator install the container-toolkit. So something (maybe symlinking `/usr/local/nvidia` to `/var/lib/nvidia`?) would be good to have in Flatcar

Reply:
Ok, returning the `/usr/local/nvidia` symlink to this sysext for now. We should investigate a more systematic approch.